app/controllers/api/base_controller.rb

```diff
@@ -59,6 +59,22 @@ def index
       render_collection(@req.subject, res, opts)
     end
 
+    def create
```

Comment:
This method should probably raise on trying to add an empty set of things, like the update version does, to keep parity (even if temporary).